### ISSUE TITLE
What are the excatly meaning for each Control Layer now?

### ISSUE THREAD

**Issue by pendave** [Acly/krita-ai-diffusion#847]
As the types of the Control Layer are increasing in the updating versions of [**krita-ai-diffusion**](https://github.com/Acly/krita-ai-diffusion), I'm a little confused when I use them.

Could you summarize it into a table?
Purpose Group | Name | Description | Effect image | Example .kra file |

Such as
Lines Control | Canny | ........................ 
Lines Control | Line Art | ........................ 
...........
Depth Control | Depth | ........................ 
...........
Refer Control | .................................

Now I'm very confused about: **Referrence, Style, Composition**, does **Referrence** equal to a visible image layer in the Layers stack?
![image](https://github.com/Acly/krita-ai-diffusion/assets/1331881/639a40f4-8fea-4c0a-9006-7f45a3635ab2)

I hope in each release there's a table notion for these:
![image](https://github.com/Acly/krita-ai-diffusion/assets/1331881/5f522bf4-151e-4308-a4a2-8651d39203af)

And Could you please add a Sorting button to a better order for those messed up Control Layers accorting to their Purpose Group?
![image](https://github.com/Acly/krita-ai-diffusion/assets/1331881/4ab8acaa-c8c3-40a1-821d-abca3888df40)

**Comment by Acly** [comment]
Reference/Style/Composition all use IP-Adapter. They act as a description like a text prompt, only using an image. You select for them a single layer with an image. Doesn't have to be visible, or the same size as your canvas. Preferrably it's square.

The rest are using ControlNet.

Yes documentation would be nice, but takes time, nobody has written any yet (since #315). Search for IP-Adapter or ControlNet you will find a lot of information and examples on the internet.

Regarding sorting: I don't typically recommend to use more than 2 or 3 of these, they cost performance and have diminishing returns.

**Comment by Acly** [comment]
https://docs.interstice.cloud/control-layers/
